packages/docs/src/routes/docs/integrations/leaflet-map/index.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: LeafletJS Map | Integrations
+keywords: 'map, interactive maps'
+contributors:
+  - mugan86
+---
+
+# LeafletJS Map
+
+Leaflet is the leading open-source JavaScript library for mobile-friendly interactive maps. 
+Weighing just about 42 KB of JS, it has all the mapping features most developers ever need.
+
+Leaflet is designed with simplicity, performance and usability in mind. 
+It works efficiently across all major desktop and mobile platforms, can be extended with lots of plugins, has a beautiful, easy to use and well-documented API and a simple, readable source code that is a joy to contribute to. [LeafletJS Map Website](https://leafletjs.com/)
+
+## Usage
+
+You can add LeafletJS Map easily by using the following Qwik starter script:
+
+```shell
+npm run qwik add leaflet-map
+```
+
+The previous command updates your app with the necessary dependencies.
+
+- `leaflet@1.9.4`
+- `@types/leaflet@1.9.4`
+
+It also adds new files inside to your project folder:
+
+- `src/helpers/boundary-box.tsx`: Check map area boundaries function.
+- `src/models/location.ts`: Model to define locations info elements to uso in props.
+- `src/models/map.ts`: Model to define map info to uso in props.
+- `src/components/leaflet-map/index.tsx`: Leaflet map simple map features component.
+- `src/routes/basic-map/index.tsx`: Example to consume Leaflet Map component with demo data
+
+## Consideration to take into account
+
+### Working with `npm` or `Yarn`:
+
+In the `src/components/leaflet-map/index.tsx` file you must have references to the CSS styles so that the maps are displayed correctly. By default it is added, but if for some reason the map looks strange when displayed, check that it is correctly referencing the CSS file within node_modules in the `leaflet` package.
+
+```bash
+node_modules/leaflet/dist/leaflet.css
+```
+
+### Working with `pnpm`:
+
+```
+node_modules/.pnpm/leaflet@1.9.4/node_modules/leaflet/dist/leaflet.css
+```
+
+## Interesting info about LeafletJS Map:
+
+### Official
+
+- `Tutorials`: Examples step by step to reference to create new features using Documentation. [Reference](https://leafletjs.com/examples.html).
+- `Docs`: All necessary info to work with LeafletJS. [Reference](https://leafletjs.com/reference.html)
+
+### Community
+
+#### English
+
+- Pending to take contributions
+
+#### Spanish
+
+- Pending to take contributions
```

Comment:
What's the purpose of this Community section?

Reply:
Add online courses, videotutorials, articles,...about Leaflet. In Spanish I have more material but my idea is recopile in English and Spanish.

Reply:
I see, let's remove this section to keep the docs clean. you can always add more resources in another PR later 💪